src/Pages/Signup/SignupInputForm/SignupInputForm.styles.scss

```
@@ -0,0 +1,239 @@
+.SignupInputForm {
+  margin-top: 19px;
+  display: flex;
+  align-items: center;
+  .input-content {
+    display: inline-block;
+    width: 159px;
+    padding-left: 20px;
+    font-weight: 700;
+    font-size: 14px;
+    color: #333;
+    line-height: 20px;
+    vertical-align: top;
+    text-align: left;
+    .signup-ico {
+      padding-right: 2px;
+      text-align: left;
+      color: #ee6a7b;
+    }
+  }
+  .signup-input {
+    width: 332px;
+    height: 44px;
+    padding: 0 14px;
+    border: 1px solid #ccc;
+    font-size: 14px;
+    color: #333;
+    line-height: 20px;
+    border-radius: 3px;
+    background: #fff;
+    outline-color: gray;
+  }
+  .signup-input-off {
+    display: none;
+  }
+  .special-birth {
+    display: flex;
+    width: 332px;
+    padding-left: 18px;
+    border: 1px solid #ccc;
+    border-radius: 3px;
+    .birth {
+      width: 84px;
+      height: 40px;
+      border: 0;
+      text-align: center;
+      outline: none;
+    }
+    .bar {
+      font-size: 14px;
+      padding: 0 9px;
+      color: #ccc;
+      line-height: 42px;
+      text-align: center;
+    }
+  }
+  .special-gender {
+    border: none;
+    padding: 0;
+    line-height: 44px;
+    .gender-check-lists {
```

```suggestion
    line-height: 44px;
    
    .gender-check-lists {
```
가독성 측면에서 새로운 셀렉터가 시작할 때는 한줄 띄워주세요

---

가독성 측면에 좀 더 신경쓰드록 하겠습니다.